templates/boltzmann/core/middleware.ts

```diff
@@ -53,16 +53,22 @@ async function buildMiddleware (middleware: MiddlewareConfig[], router: Handler)
     isDev()
     ? (mw: Middleware) => [
       // {% if honeycomb %}
-      honeycombMiddlewareSpans(mw),
+      startSpan(mw),
       // {% endif %}
       dev(mw.name),
-      enforceInvariants()
+      enforceInvariants(),
+      // {% if honeycomb %}
+      endSpan(mw),
```

Comment:
The bug is that endSpan needs to be spliced after the mw, not before - but finishing this thought will require refactoring how it terminates at the router.